js-weather/js/script.js

```diff
@@ -0,0 +1,67 @@
+let xhr = new XMLHttpRequest();
+let AppId = '4b5774e9f3d2a07b84f0f2f88e486224';
+
+document.addEventListener('DOMContentLoaded', function () {
+    //デフォルトロンドン
+    let LondonUrl = "http://api.openweathermap.org/data/2.5/weather?q=London&appid=4b5774e9f3d2a07b84f0f2f88e486224";
+    xhr.open('GET', LondonUrl, true);
+    xhr.send();
+    //通信ステータスが変わったら実行される関数
+    xhr.onreadystatechange = function () {
+        //通信が完了したらShowTodaysWeather関数を実行
+        if (xhr.readyState == 4) {
+            TodaysWeather(xhr.responseText);
+        }
+    }
+    // id、weatherが変わったら↓
+    document.getElementById('weather').addEventListener('change', function () {
+        let CityName = this.value;
+        let requestUrl = "http://api.openweathermap.org/data/2.5/weather?q=" + CityName + "&appid=4b5774e9f3d2a07b84f0f2f88e486224";
+        xhr.open('GET', requestUrl, true);
+        xhr.send();
+        //通信ステータスが変わったら実行される関数
+        xhr.onreadystatechange = function () {
+            //通信が完了したらShowTodaysWeather関数を実行
+            if (xhr.readyState == 4) {
+                TodaysWeather(xhr.responseText);
+            }
+        }
+    });
+});
+
+function TodaysWeather(response) {
```

Comment:
ここもアロー関数にする

```suggestion
coinst todaysWeather = (response) => {
...
};
```
こうします。